src/components/UserProfile/UserProfileAdmin/index.js

```diff
@@ -7,25 +7,49 @@ import TableBody from "@material-ui/core/TableBody";
 import TableCell from "@material-ui/core/TableCell";
 import TableHead from "@material-ui/core/TableHead";
 import TableRow from "@material-ui/core/TableRow";
-import { loaderActions } from "../../../Redux/actionCreators";
+import { loaderActions, rfaiContractActions } from "../../../Redux/actionCreators";
 import { LoaderContent } from "../../../utility/constants/LoaderContent";
 import { useStyles } from "./styles";
 import StyledButton from "../../common/StyledButton";
 import AlertBox, { alertTypes } from "../../common/AlertBox";
 
 import { waitForTransaction, addOrUpdateFoundationMembers } from "../../../utility/BlockchainHelper";
+import { fromWei } from "../../../utility/GenHelperFunctions";
 import web3 from "web3";
 
-const UserProfileAdmin = ({ classes, metamaskDetails, foundationMembers, startLoader, stopLoader }) => {
+const UserProfileAdmin = ({
+  classes,
+  metamaskDetails,
+  foundationMembers,
+  startLoader,
+  stopLoader,
+  rfaiMinStake,
+  rfaiMaxStakers,
+  rfaiOwner,
+  updateRFAIOwner,
+  updateRFAIMinStake,
+  updateRFAIMaxStakers,
+}) => {
   const [alert, setAlert] = useState({ type: alertTypes.ERROR, message: undefined });
   const [foundationMember, SetFoundationMember] = useState("");
   const [memberRole, SetMemberRole] = useState(0); //Normal User, 1-Admin User who can add other members
 
+  // eslint-disable-next-line no-unused-vars
+  const [owner, SetOwner] = useState("0x0");
   // eslint-disable-next-line no-unused-vars
   const [minStake, SetMinStake] = useState(1);
   // eslint-disable-next-line no-unused-vars
   const [maxStakers, SetMaxStakers] = useState(1);
```

Comment:
why to use unused vars here?.. if all the states are used, then the `eslint-disable` could be removed.

Reply:
This future enhancements especially admin functionality.